intelmq/bots/experts/jinja/expert.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: 2021 Birger Schacht
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from intelmq.lib.bot import Bot
+
+import pathlib
+import os
+
+try:
+    from jinja2 import Template, TemplateError
+except ImportError:
+    Template = None
+
+
+class JinjaExpertBot(Bot):
+    """
+    Modify the message using the Jinja templating engine
+    Example:
+    fields:
+      output: The provider is {{ msg['feed.provider'] }}!
+      feed.url: "{{ msg['feed.url'] | upper }}"
+      extra.somejinjaoutput: file:///etc/intelmq/somejinjatemplate.j2
+    """
+
+    fields: dict = {}
```

Comment:
```suggestion
    fields: typing.Dict[str, str] = {}
```

Reply:
That won't be possible, because later we overwrite the second string with a `jinja2.Template`

Reply:
I used a `Union`